ctx.go

```diff
@@ -423,11 +426,22 @@ func body[B any](c ContextNoBody) (B, error) {
 	case "application/x-www-form-urlencoded", "multipart/form-data":
 		body, err = readURLEncoded[B](c.Req, c.readOptions)
 	case "application/xml":
-		return readXML[B](c.Req.Context(), c.Req.Body, c.readOptions)
+		body, err = readXML[B](c.Req.Context(), c.Req.Body, c.readOptions)
 	case "application/x-yaml":
-		return readYAML[B](c.Req.Context(), c.Req.Body, c.readOptions)
+		body, err = readYAML[B](c.Req.Context(), c.Req.Body, c.readOptions)
 	case "application/json":
 		fallthrough
+	case "application/octet-stream":
+		// Read c.Req Body to bytes
```

Comment:
It may seem like overkill, but I really like the how we've been using helper functions for this. How about a helper function here too?

Just a nitpick feel free to close.

Reply:
I'm just doing it as a way to harmonize the serialization within the Fuego controllers.

As it is really easy to do it with std lib, people using Fuego helper functions might better use directly the std lib I guess.